src/amo/pages/Guides/index.js

```diff
@@ -364,7 +394,7 @@ export class GuidesBase extends React.Component<InternalProps> {
   render() {
     const { clientApp, i18n, match } = this.props;
     const { slug } = match.params;
-    const content = getContent(slug, i18n);
+    const content = getContent({ slug, i18n, sections: this.props.sections });
```

Comment:
Is that needed for testing purpose?

Also, you could extract `sections` two lines above (with the other props).

Reply:
> Is that needed for testing purpose?

yes


> Also, you could extract sections two lines above (with the other props).

sure